content/rs/administering/access-control/password-rotation.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: Database Password Rotation
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+To maintain secure access to your Redis Enterprise Software (RS) cluster and databases,
+most organizational security policies require that you rotate user passwords on a regular basis.
+
+For user access to the RS web UI,
+you can set a [password expiration policy]({{< relref "/rs/administering/access-control/#setting-local-user-password-expiration" >}}) that prompts the user to change their password.
+However, for database connections that rely on password authentication,
+you need to allow for authentication with the existing password while you rollout the new password to your systems.
+
+With the RS REST API, you can add additional passwords to an RS user account for authentication to the database or the Admin Console and API.
+After the old password is replaced in the database connections,
+just delete the old password to finish the password rotation process.
+
+{{ warning }}
+If you reset the password for a user in the RS Admin Console,
+that password replaces all other passwords for the user.
+{{ /warning }}
+
+## Adding an additional password to an RS user account
+
+You can add an additional password to a user account as long as:
+
+- The new password does not already exist as a password for the user
+- If [password complexity]({{< relref "/rs/administering/access-control/#setting-up-local-password-complexity" >}}) is required, the new password meets the complexity requirements
```

Comment:
Maybe: "The new password meets any [complexity requirements] (if enabled)

The first bullet here naturally completes the preceding sentence, but the second doesn't because of the way it's phrased. You can easily rephrase it so that it grammatically completes the sentence that introduces these bullets. This make for a smoother reading experience.

Reply:
Didn't see this comment before. Got it.